### PR TITLE
uninstall dev dependencies in travis

### PR DESCRIPTION
**Pull request by thekp**
Resolves: n/a

**Overall change:** _Mimic the production build by adding npm prune --production to the 2nd test stage in travis_

**no testing required, as it is a change to our travis only**

**Code changes:**

- _add npm prune --production to uninstall dev deps_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Review by oluoluoxenfree — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -34,6 +34,7 @@ jobs:
       script:
         - npm run build
         - xvfb-run -a npm run test:prod:travis
+        - npm prune --production
```
Comment (dr3) [inline .travis.yml:37]: Does this do anything? Im pretty sure this is the end of the stage?

**Review by dr3 — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (thekp) [reply .travis.yml:37]: > it's also important for when jenkins is down that we can trust the travis build

Comment (dr3) [reply .travis.yml:37]: >Data persistence between stages and jobs
>
>It is important to note that jobs do not share storage, as each job runs in a fresh VM or container. If your jobs need to share files (e.g., using build artifacts from the “Test” stage for deployment in the subsequent “Deploy” stage), you need to use an external storage mechanism such as S3 and a remote scp server.

https://docs.travis-ci.com/user/build-stages/#data-persistence-between-stages-and-jobs

Comment (jamesbhobbs) [reply .travis.yml:37]: It doesn't do anything at all at the moment, I think it should be done between build and the tests to mimic the live deploy and prune what won't be there in production.

Comment (dr3) [reply .travis.yml:37]: @thekp I think your `npm prune --production` needs to be moved up a line in between build and `npm run test:prod:travis`

But I think benjamins doing this here https://github.com/bbc/simorgh/pull/2905/files

**Review by dr3 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**